go-chaos/cmd/backup.go

```diff
@@ -58,12 +71,121 @@ var waitForBackupCommand = &cobra.Command{
 	RunE:  waitForBackup,
 }
 
-func takeBackup(cmd *cobra.Command, args []string) error {
+var restoreBackupCommand = &cobra.Command{
+	Use:   "restore",
+	Short: "Restore from a given backup id",
+	RunE:  restoreFromBackup,
+}
+
+func setupBackup(cmd *cobra.Command, _ []string) error {
+	k8Client, err := internal.CreateK8Client()
+	if err != nil {
+		panic(err)
+	}
+
+	namespace := k8Client.GetCurrentNamespace()
+
+	err = k8Client.PauseReconciliation()
```

Comment:
:question: : What happens on our clusters here?

Reply:
Set's the `pauseReconciliation` flag to true which prevents the SaaS operator from reverting our changes to the stateful sets.

Reply:
But I have seen later you ignore it. But was not sure when reading first.